cmd/apmtool/tracegen.go

```diff
@@ -38,10 +41,34 @@ func (cmd *Commands) sendTrace(c *cli.Context) error {
 	if err != nil {
 		return err
 	}
-	apmTracer, err := apm.NewTracer(newUniqueServiceName("service", "intake"), "0.0.1")
+
+	// TODO: the tracer should be created by tracegen from tracegen.Config
```

Comment:
Could you clarify what this means?

Reply:
Seeing as I was making some changes anyway, I have addressed this in 996120ee70bd0f50ae906f9411894b5e90efaa94.